data/Modes.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "name": "Prism",
+        "instrument": "SpeX",
```

Comment:
Does case matter/ break anything? e.g. "spex" == "SpeX"?

Reply:
I believe this is case insensitive, but worth double checking.